dev/packages/alpha/netflow-0.0.2/dataset/log/agent/stream/stream.yml

```diff
@@ -0,0 +1,21 @@
+input: netflow
+protocols: [v1, v5, v6, v7, v8, v9, ipfix]
+host: '{{this.netflow_host}}:{{this.netflow_port}}'
+max_message_size: '{{this.max_message_size}}'
+expiration_timeout: '{{this.expiration_timeout}}'
+queue_size: {{this.queue_size}}
+{{#if this.timeout}}
```

Comment:
I'm stumbling over the `this` here. @mtojek do you know if this is necessary?

Reply:
This is already fixed, even though both forms (with/without `this`) are correct.

The previous implementation of processing stream configuration left `this` for consistency. If you upgrade to the new implementation, you will see a better formed file: https://github.com/elastic/package-registry/blob/master/dev/packages/beats/netflow-0.0.1/dataset/log/agent/stream/netflow.yml.hbs